src/bundle.js

```diff
@@ -0,0 +1,45 @@
+import './assets/stylesheets/style.scss';
+import { creator, createModal } from './modules/helpers';
+import { filterData, getWeather } from './modules/data';
+import { createWeatherDisplay, setValues, symbols } from './modules/display';
+import { formSearch, units } from './modules/form';
+import { createBGContainer, setBGImage, iconList } from './modules/background';
+
+const content = document.getElementById('content');
+const main = creator(content, 'main', 'append');
+formSearch(main, units);
+createBGContainer(main);
+createWeatherDisplay(main);
+
+const options = {
+  enableHighAccuracy: true,
+  timeout: 5000,
+  maximumAge: 0,
+};
+
+function manageResponse(weatherData) {
+  if (weatherData) {
+    const allData = filterData(weatherData);
+    setBGImage(iconList, allData[6]);
+    setValues(symbols, ...allData);
+  } else {
+    const modalContent = createModal(main);
+    modalContent.innerHTML = 'The city provided could not be found.';
+    document.getElementById('error-msg').style.display = 'block';
```

Comment:
Let's separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/